cg/cli/workflow/microsalt/base.py

```diff
@@ -152,12 +158,10 @@ def run(
             "--input",
             Path(fastq_path, sample_id).absolute().as_posix(),
         ]
-    analysis_api.process.run_command(parameters=analyse_command, dry_run=dry_run)
 
     if sample_id or dry_run:
+        analysis_api.process.run_command(parameters=analyse_command, dry_run=dry_run)
```

Comment:
Could you add a comment here? Explain why would you execute the command always if dry_run is True

Reply:
Dry-run argument is still being passed. So its not running for real.

Reply:
I guessed that, just think it would be nice with a clarifying comment